app/src/main/java/com/lastaoutdoor/lasta/ui/screen/moreinfo/MoreInfoScreen.kt

```diff
@@ -52,7 +52,7 @@ fun MoreInfoScreen(
       // contains the top icon buttons
       item { TopBar(navigateBack) }
       // displays activity title and duration
-      item { ActivityTitleZone(activityToDisplay) }
+      item { ActivityTitleZone(activityToDisplay, processDiffText) }

```

Comment:
You don't need to pass lambda, you can use the enum defined functions, either .name or toString()